README-Gitpod.md

```diff
@@ -0,0 +1,39 @@
+Token Contract in AssemblyScript - Gitpod version
+=================================================
+
+This is a stripped-down example where we'll demonstrate creating a token. We'll visit a page, sign in and use your browser's console to run a commands to initialize, send, and get the balance of a custom token.
+
+This README is specific to Gitpod and this example. For local development, please see [README.md](README.md).
+
+## Description
+
+This project contains implementation of token contract similar to [ERC20](https://theethereum.wiki/w/index.php/ERC20_Token_Standard).
+
+## Getting started
```

Comment:
While we're here, let's standardize heading style. I would prefer we use h1 tags for these headings, same as the main heading of the file. While I can see a semantic argument for keeping the main document name at a higher level than these other headings, I think both the HTML & markdown versions of the document end up cleaner if we use h1 for these.

Then, for all h1, we can use the `=`-underlined style, and for all h2, we can use the `-`-underlined style:

```markdown
Getting Started
===============

There are a couple ways to do this, the quick way and the complete way.


The Quick Way
-------------

lorem ipsum


The Complete Way
----------------

dolor sit amet
```

Reply:
k